system-test/dns.ts

```diff
@@ -115,9 +115,26 @@ describe('dns', () => {
   };
 
   before(async () => {
+    // Clean up any leaked resources
     const [zones] = await dns.getZones();
-    await Promise.all(zones.map(zone => zone.delete({force: true})));
```

Comment:
@bcoe this is kind of funny.  If two test suites happened to be running at the same time, there's no way this was going to work 😆

Reply:
Good catch 👍